plugins/nv-filters/nvvfx-load.h

```diff
@@ -252,7 +252,7 @@ typedef struct
 #ifdef _MSC_VER
 	__declspec(dllexport)
 #endif // _MSC_VER
-		NvCVImage {
+	NvCVImage {
```

Comment:
Is this file vendored @pkviet ?

Reply:
Hmm. Both `__declspec(dllexport)` and `NvCVImage {` could probably be fully unindented, but it probably doesn't matter.